dev-java/hamcrest-core/hamcrest-core-1.3-r2.ebuild

```diff
@@ -37,24 +36,26 @@ PATCHES=(
 
 src_prepare() {
 	default
-	# Empty out the contents of the generator target; it has already been built.
-	eapply "${FILESDIR}/hamcrest-1.3-empty_generator.patch"
+	find -iname "*.jar" -exec rm -v {} + || die "Unable to clean bundled JAR files"
```

Comment:
[this](https://github.com/gentoo/gentoo/blob/master/eclass/java-utils-2.eclass#L2925) is the standard way to do it.

Reply:
> [this](https://github.com/gentoo/gentoo/blob/master/eclass/java-utils-2.eclass#L2925) is the standard way to do it.

Are you saying that's also to be removed?  Then I could drop the whole src_prepare() phase ...
But later, gonna take a break now

Reply:
@vaukai i understand, this ebuild is exhausting... :-) but i guess you learned a lot :-) anyway, to your question, you should call `java-pkg_clean` from `src_prepare`.

Reply:
Yes, done.
Also mentioned here: https://wiki.gentoo.org/wiki/Java_Developer_Guide#Removing_bundled_classes_and_jars